openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -83,6 +87,21 @@ def setUp(self):
 
         self.url = reverse("accounts_api", kwargs={'username': self.user.username})
 
+    def _verify_profile_image_data(self, data):
+        """
+        Verify the profile image data in a GET response for self.user is
+        what we expect.  Note that we mocked out the URL generation
```

Comment:
By "what we expect", this is always checking that the user does not have a custom profile_image. Either rename the method so it is clear (and update doc), or have the method handle the other case as well. I assume someplace we are going to test what is returned when the user does have a custom profile image set.